src/docs/product/integrations/issue-tracking/height/index.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Height
+sidebar_order: 1
+description: "Learn more about Sentry's Height integration and how you can effortlessly create new Height issues from Sentry and link Sentry issues to existing Height tasks."
+---
+
+The Height Sentry integration allows your team to record and quickly resolve critical errors by linking new or existing Height tasks to Sentry issues. Linking tasks gives your teammates visibility into updates from Sentry, and can automatically mark Sentry issues as __Resolved__ as the task status changes in Height. Additionally, set up custom alerts to create new Height tasks when key triggers you care about, like reported errors or session crashes, spike above a certain threshold.
+
+Height needs to be set up only once per organization, and then it is available for _all_ projects. It's maintained and supported by the company that created it. For more details, see [Integration Platform](/product/integrations/integration-platform/).
```

Comment:
Eugh so I checked and we actually have this first sentence in 14 other pages - it's an artifact of explaining how first party integrations are better than plugins, but since integrations are the norm I'd lean towards removing it entirely across the 16 pages it appears in (in a separate PR), since the plugins (which need to be enabled per project) explicitly call it out. What do you think @imatwawana (also @brianthi)?

Reply:
That makes sense to me

Reply:
makes sense to me too

Reply:
Cool, I removed it from the 2 files I'm editing here already but will make a separate PR to remove it from the remaining pages.